keps/sig-cloud-provider/azure/20180711-azure-availability-zones.md

```diff
@@ -156,7 +156,7 @@ Note that
 
 ### PVLabeler interface
 
-To setup AzureDisk's zone label correctly (required by cloud-controller-manager's PersistentVolumeLabelController), Azure cloud provider's [PVLabeler](https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/cloud.go#L212) interface should be implemented:
+To setup AzureDisk's zone label correctly (required by cloud-controller-manager's PersistentVolumeLabelController), Azure cloud provider's [PVLabeler](https://github.com/kubernetes/cloud-provider/blob/master/cloud.go#L212) interface should be implemented:
```

Comment:
Using line offsets is hard to keep upto date. Note sure if we can find a better way. Right now the correct offset is 254. So the link should be https://github.com/kubernetes/cloud-provider/blob/master/cloud.go#L254